### PR TITLE
New tutorial for Orleans deployment to Azure App Service

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

New tutorial for Orleans deployment to Azure App Service. Based on https://github.com/Azure-Samples/Orleans-Cluster-on-Azure-App-Service

Part of #28672

## Internal preview

Tagging @ReubenBond and @bradygaster 

- [Deploy Orleans to Azure App Service](https://review.docs.microsoft.com/en-us/dotnet/orleans/deployment/deploy-to-azure-app-service?branch=pr-en-us-29083)

### CODE REVIEW

**Comment by bradygaster** [comment]
ship it!

**Review by tdykstra — APPROVED** [review]
Looks great, I left mainly grammar and style suggestions.

File: docs/orleans/deployment/deploy-to-azure-app-service.md [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+> [!div class="checklist"]
+>
+> - Deploy an Orleans application to Azure App Service
+> - Automate deployment using GitHub Actions and Azure Bicep
+> - Configure the virtual network for the app within Azure
+
+## Prerequisites
+
+- A [GitHub account](https://github.com/join)
+- The [.NET 6 SDK or later](https://dotnet.microsoft.com/download/dotnet)
```
Comment (IEvangelist) [inline docs/orleans/deployment/deploy-to-azure-app-service.md:31]:
```suggestion
- The [.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet)
```

File: docs/orleans/deployment/deploy-to-azure-app-service.md [hunk 2]
```diff
@@ -0,0 +1,480 @@
...
+>
+> - Deploy an Orleans application to Azure App Service
+> - Automate deployment using GitHub Actions and Azure Bicep
+> - Configure the virtual network for the app within Azure
+
+## Prerequisites
+
+- A [GitHub account](https://github.com/join)
+- The [.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet)
+- The [Azure CLI](/cli/azure/install-azure-cli)
```
Comment (IEvangelist) [inline docs/orleans/deployment/deploy-to-azure-app-service.md:32]:
```suggestion
- A [GitHub account](https://github.com/join)
- [Read an introduction to Orleans](../overview.md)
- The [.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet)
- The [Azure CLI](/cli/azure/install-azure-cli)
```

File: docs/orleans/deployment/deploy-to-azure-app-service.md [hunk 3]
```diff
@@ -0,0 +1,480 @@
...
+
+## Inside the shopping cart app
+
+Orleans is a reliable and scalable framework for building distributed applications. For this tutorial, you will deploy a simple shopping cart app built using Orleans to Azure App Service. The app exposes the ability to manage inventory, add and remove items in a cart, and shop available products. The client is built using Blazor with a server hosting model. The app is architected as follows:
+
+:::image type="content" source="media/shopping-cart-app-arch.png" lightbox="media/shopping-cart-app-arch.png" alt-text="Orleans: Shopping Cart sample app architecture.":::
+
+The preceding diagram shows that the client is the server-side Blazor app. It's composed of several services that consume a corresponding Orleans grain. Each service pairs with an Orleans grain as follows:
```
[the remark this answers is not in the document]
Comment (IEvangelist) [reply docs/orleans/deployment/deploy-to-azure-app-service.md:52]: I added it to the prerequisites.

File: docs/orleans/deployment/deploy-to-azure-app-service.md [hunk 4]
```diff
@@ -0,0 +1,480 @@
...
+The app will need to be packaged for deployment. In the `Orleans.ShoppingCart.Silos` project, we define a `Target` element that runs after the `Publish` step. This will zip the publish directory into a _silo.zip_ file:
+
+```xml
+<Target Name="ZipPublishOutput" AfterTargets="Publish">
+    <Delete Files="$(ProjectDir)\..\silo.zip" />
+    <ZipDirectory SourceDirectory="$(PublishDir)" DestinationFile="$(ProjectDir)\..\silo.zip" />
+</Target>
+```
+
+There are many ways to deploy a .NET app to Azure App Service. In this tutorial, you use GitHub Actions, Azure Bicep, and the .NET and Azure CLIs.
```
Comment (IEvangelist) [inline docs/orleans/deployment/deploy-to-azure-app-service.md:177]:
```suggestion
There are many ways to deploy a .NET app to Azure App Service. In this tutorial, you use GitHub Actions, Azure Bicep, and the .NET and Azure CLIs. Consider the _./github/workflows/deploy.yml_ file in the root of the GitHub repository:
```

File: docs/orleans/deployment/deploy-to-azure-app-service.md [hunk 5]
```diff
@@ -0,0 +1,480 @@
...
+```xml
+<Target Name="ZipPublishOutput" AfterTargets="Publish">
+    <Delete Files="$(ProjectDir)\..\silo.zip" />
+    <ZipDirectory SourceDirectory="$(PublishDir)" DestinationFile="$(ProjectDir)\..\silo.zip" />
+</Target>
+```
+
+There are many ways to deploy a .NET app to Azure App Service, in this tutorial we'll use GitHub Actions, Azure Bicep, and the .NET and Azure CLIs.
+
+```yml
```
[the remark this answers is not in the document]
Comment (IEvangelist) [reply docs/orleans/deployment/deploy-to-azure-app-service.md:179]: Add the file location.

**Pull request merged**